.github/workflows/publish.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  publish:
+    permissions:
+      id-token: write # Required for authentication using OIDC
+    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
```

Comment:
Could we pin to the most recent PATCH version (which seems to be `1.6.0` according to the https://github.com/dart-lang/setup-dart/releases)?

Reply:
Done in https://github.com/ButterCMS/buttercms-dart/pull/9/commits/1320caad41fb0d8e8e11a6c49ff0df86d1367007